homeassistant/components/hypontech/sensor.py

```diff
@@ -0,0 +1,98 @@
+"""The read-only sensors for Hypontech integration."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from hyponcloud import OverviewData
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import UnitOfEnergy, UnitOfPower
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import HypontechConfigEntry, HypontechDataCoordinator
+
+
+@dataclass(frozen=True, kw_only=True)
+class HypontechSensorDescription(SensorEntityDescription):  # type: ignore[override]
```

Comment:
```suggestion
class HypontechSensorDescription(SensorEntityDescription):
```

Reply:
I got this old error removing it https://github.com/home-assistant/core/pull/135589. I'm figuring out why.